### PR TITLE
#64 quick summary of task

### PR DESCRIPTION
**Pull request by AXhing-LLW**

### CODE REVIEW

File: src/components/main/StaffList.js [hunk 1]
```diff
@@ -44,19 +44,33 @@ class StaffList extends React.Component {
...
         const staffList = this.props.staffs.map((staff, i) =>
             <tr key={staff.id} id={staff.id} >
                 <td>{staff.type}</td>
                 <td>
-                    {staff.quantity}  
-                    <span className="btext" onClick={this.listAdd.bind(staff,i)}>+</span> 
-                    <span className="btext2" onClick={this.listSub.bind(staff,i)}>-</span>
+                    {staff.quantity} 
+                    <button type="button" className="btn btn-outline-primary" onClick={this.listAdd.bind(staff, i)}>&#10133;</button>
+                    <button type="button" className="btn btn-outline-primary" onClick={this.listSub.bind(staff, i)}>&#10134;</button>
```
Comment (beckaseevers) [inline src/components/main/StaffList.js:66]: I'm not sure why you did this, because this is not related to your #64 task. This is a task that Emma is going to work on. You can leave it for now, but it'll be changed by Emma, just so you know. Please

File: public/manifest.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "short_name": "React App",
+  "short_name": "Nursing Sim",
```
[the remark this answers is not in the document]
Comment (beckaseevers) [reply public/manifest.json:2]: Also, when I did an npm start on your branch, the React logo is still visible on the tab. This should be changed in manifest.json as well to be the BSU logo

File: public/index.html [hunk 3]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Nursing Sim</title>
```
Comment (beckaseevers) [inline public/index.html:27]: Change to "BSU Nurse Staffing Sim"

**Review by beckaseevers — CHANGES_REQUESTED** [review]
It looks like you did work that was Emma's task and didn't finish everything for #64 (the logo on the browser tab shouldn't be the React logo). You can leave the changes you made for the edit and delete icons, but Emma will be changing these since she has already been working on that task.

File: src/components/main/StaffList.js (hunk 1 above)
Comment (AXhing-LLW) [reply src/components/main/StaffList.js]: I fix it, and I'm sorry to bother you!

**Review by beckaseevers — APPROVED** [review]
